src/tls.c

```diff
@@ -151,6 +151,9 @@ void tlsInit(void) {
 
     /* Server configuration */
     server.tls_auth_clients = 1;    /* Secure by default */
+    server.tls_ctx_config.session_caching = 1;
+    server.tls_ctx_config.session_cache_size = 20*1024;
+    server.tls_ctx_config.session_cache_timeout = 300;
```

Comment:
these are actually not needed, initServerConfig puts the defaults from config.c, unless they're not handled by the common infrastructure.
let's delete these 4 lines so that we don't get a potential diversion between the defaults defined in two places..